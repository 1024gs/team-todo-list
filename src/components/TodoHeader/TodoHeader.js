import React from 'react';
import './TodoHeader.scss';

const TodoHeader = ({title, subtitle}) => {
  return (
    <div className="TodoHeader">
      <div className="no-collapsing-margins">
        <div className="row">
          <div className="col s12">
            <h1>{title}</h1>
            <p>{subtitle}</p>
          </div>
        </div>
      </div>
    </div>
  );
};

export default TodoHeader;
