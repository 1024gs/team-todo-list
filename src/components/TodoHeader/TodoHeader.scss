@import '../../vars';

.TodoHeader {
  color: #fff;
  background-color: $primary-color;
  background-image: url(../../../public/the-wave-coyote-buttes.jpg);
  background-blend-mode: soft-light;
  background-size: cover;
  background-repeat: no-repeat;
  background-position: top, center;
  padding-top: 30px;
  padding-left: 9px;
  padding-right: 9px;
}

.TodoHeader h1 {
  font-size: $h4-fontsize;
  font-weight: 300;
  margin-bottom: 0;
}
