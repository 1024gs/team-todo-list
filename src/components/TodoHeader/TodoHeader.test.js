import React from 'react';
import TodoHeader from './TodoHeader';
import {shallow} from 'enzyme';

const setup = (propOverrides) => {
  const props = Object.assign(
    {title: 'Team To-do List', subtitle: 'It might be a date'},
    propOverrides
  );
  const wrapper = shallow(<TodoHeader {...props} />);

  return {
    props,
    wrapper
  };
};

describe('TodoHeader', () => {
  it('renders', () => {
    const {wrapper} = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
