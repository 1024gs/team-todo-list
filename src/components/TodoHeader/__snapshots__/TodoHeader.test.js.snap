// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoHeader renders 1`] = `
<div
  className="TodoHeader"
>
  <div
    className="no-collapsing-margins"
  >
    <div
      className="row"
    >
      <div
        className="col s12"
      >
        <h1>
          Team To-do List
        </h1>
        <p>
          It might be a date
        </p>
      </div>
    </div>
  </div>
</div>
`;
